laa_crime_application_store_app/services/permissions.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+from sentry_sdk import capture_message
+
+from laa_crime_application_store_app.models.application_schema import Application
+
+
+class Permissions:
+    LOCKED_STATES = ("granted", "expired", "auto_grant")
+    CASEWORKER_EDITABLE_STATES = ("submitted", "provider_updated")
```

Comment:
This needs to include `sent_back` because the caseworker app is responsible for updating `sent_back` applications to `expired` via a daily scheduled job.

Reply:
so this break out rule of only one app can edit in each state... :(